docs/src/content/docs/commands/HVALS.md

```diff
@@ -82,7 +82,7 @@ If the hash `user:1003` does not exist:
 - `HGETALL`: The `HGETALL` command retrieves all field-value pairs in a hash, providing both names and values.
 
 ## Notes
-
+- Please refer to the [HGETALL](/commands/hgetall) command documentation for more information.
```

Comment:
same as above. Lets add the information in Alternatives section and style as discussed above.